src/intervals/conversion.jl

```diff
@@ -30,15 +24,8 @@ function convert{T<:AbstractFloat, S<:Real}(::Type{Interval{T}}, x::S)
     # use @round_up and @round_down here?
 end
 
-function convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64)
-    II = convert(Interval{T}, rationalize(x))
-    # This prevents that rationalize(x) returns a zero when x is very small
-    if x != zero(x) && II == zero(Interval{T})
-        return Interval(parse(T, string(x), RoundDown),
-                        parse(T, string(x), RoundUp))
-    end
-    II
-end
+convert{T<:AbstractFloat}(::Type{Interval{T}}, x::Float64) =
```

Comment:
Unfortunately the conversion to and from a string is very slow, that's what I'm pointing out. (And probably `rationalize` was too.)